### PR TITLE
feat: add scaffolder actions plugins

### PR DESCRIPTION
**Pull request by tumido**
## Description

Add scaffolder actions for quay, regex, sonarqube, servicenow

Per https://redhat-internal.slack.com/archives/C05HGAR2DT5/p1700654673164509?thread_ts=1700250550.958579&cid=C05HGAR2DT5

## Which issue(s) does this PR fix

fixes https://github.com/janus-idp/backstage-showcase/issues/629
## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

### CODE REVIEW

File: dynamic-plugins.default.yaml [hunk 1]
```diff
@@ -495,6 +495,18 @@ plugins:
   # Standalone plugins
   - package: ./dynamic-plugins/dist/roadiehq-scaffolder-backend-module-utils-dynamic
 
+  - package: ./dynamic-plugins/dist/janus-idp-backstage-scaffolder-backend-module-quay
```
[the remark this answers is not in the document]
Comment (tumido) [reply dynamic-plugins.default.yaml:498]: fixed

File: dynamic-plugins.default.yaml [hunk 2]
```diff
@@ -495,6 +495,18 @@ plugins:
...
+    disabled: true
+
+  - package: ./dynamic-plugins/dist/janus-idp-backstage-scaffolder-backend-module-regex
+    disabled: true
+
+  - package: ./dynamic-plugins/dist/janus-idp-backstage-scaffolder-backend-module-servicenow
```
[the remark this answers is not in the document]
Comment (tumido) [reply dynamic-plugins.default.yaml:502]: updated

**Review by davidfestal — APPROVED** [review]

**Review by nickboldt — APPROVED** [review]
no need to configure regex and quay plugins in dynamic-plugins.default.yaml? If not, then +1.

But I notice  `janus-idp-backstage-scaffolder-backend-module-sonarqube` is set to disabled. Are these other two enabled by default?  Should we be explicit with `disabled: false` ?

**Comment by tumido** [comment]
> no need to configure regex and quay plugins in dynamic-plugins.default.yaml? If not, then +1.

Correct. Regex and quay do not have any `app-config.yaml` settings available therefore we do not need to provide `pluginConfig` for them.

> But I notice janus-idp-backstage-scaffolder-backend-module-sonarqube is set to disabled. Are these other two enabled by default? Should we be explicit with disabled: false ?

Correct, since quay and regex do not require any settings, they are enabled by default as per https://docs.google.com/spreadsheets/d/1dNUAGTeosEzreX9fxFTW1Vq3oGCbHroZ_DFEL8sjfjY/edit?disco=AAABBIH-PLE

The other 2 are disabled, since they require config values to be provided by user.

**Pull request merged**